template_feed/Microsoft.DotNet.Test.ProjectTemplates.8.0/content/NUnit-FSharp-Item/.template.config/en-US/strings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "1.0.0.0",
+  "strings": {
+    "name": "NUnit Test Project",
+    "description": "A project that contains NUnit tests that can run on .NET Core on Windows, Linux and MacOS."
```

Comment:
Shall we keep the `.NET Core` naming? I thought that the idea was to call .NET 5+ only `.NET`?

Reply:
Updated in all templates since net5 (including).